CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## XX.XX.XX - 2023-XX-XX
+* [CHANGED][8231](https://github.com/stripe/stripe-android/pull/8231) Separator text under Link/Google Pay button from "Or pay using"/"Or pay with a card" to "Or use"/"Or use a card" when using SetupIntent.
```

Comment:
This doesn‘t quite seem release-notes-worthy to me. Are you ok with removing this?

Reply:
Yeah that's fine with me! I added it because iOS did for a similar change [here](https://github.com/stripe/stripe-ios/pull/3231/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)

Reply:
Ah, gotcha. If you’re keeping it, then I’d suggest to put it under a `### PaymentSheet` subtitle.

Reply:
Updated 👍